centers/models/center.py

```diff
@@ -14,8 +14,7 @@ class Center(models.Model):
     address = models.CharField(max_length=255, blank=True, null=True)
     phone = models.CharField(max_length=45, blank=True, null=True)
     url = models.URLField(blank=True, null=True)
-    hash_id = models.CharField(
-        max_length=12, unique=True)
+    hash_id = models.CharField(max_length=12, unique=True)
```

Comment:
and another case 

http://stackoverflow.com/questions/4964197/converting-a-number-base-10-to-base-62-a-za-z0-9

you can try base62 or others for shorten urls